CONTRIBUTING.md

```diff
@@ -57,7 +57,7 @@ The workflow is as follows for a **new** protocol:
 
 1.  make sure your local clone of the remote repository is up to date:
 
-    1.  with the master branch checked out, press the pull button in the Git pane
+    1.  with the main branch checked out, press the pull button in the Git pane
```

Comment:
In the near future I'd recommend to use `checklist::new_branch()` to create a new branch. Will be available once https://github.com/inbo/checklist/pull/54 is merged.

Reply:
checklist::new_branch() is now available

Reply:
Done in 7217078